### PR TITLE
Overhaul 4: The README's Last Hurrah

### PR DESCRIPTION
**Pull request by KendrickLamar2003**
- Added LTS notes
- Added mentions of the Quest 3
- Added ModernFix, ServerCore, Cull Less Leaves, and Fabric Language Kotlin as per devmods.json
- Removed Renderers
- Reworked Mod-related crash solutions
- Added a strong warning NOT to enable Seated Mode
- Added README contributors
- Fixed many grammatical errors

Comments and edits are welcomed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,100 +1,113 @@
...
+Secondly, install the QuestCraft APK to your VR Headset using **[SideQuest](https://sidequestvr.com/setup-howto)** and follow the necessary steps to connect your Quest. Drag and drop the APK, and wait for it to install.
 
-**NOTE: You may encounter crashes when downloading your selected MC version, ESPECIALLY on v53 and later. This is because Meta can't fix memory leaks, for some reason. Don't fret; launch the app again, press play again, and the download will continue where it left of before the crash. If it continues, please keep trying. If, after 4 attempts, you still are not loaded in, please visit the support channel of the QuestCraft Discord (discord.gg/questcraft)**
+Next, open your headset's QuestCraft launcher (through Unknown Sources). Be sure to grant all requested permissions (Microphone perms are for Simple Voice Chat!). Once you're done, you'll follow the on-screen prompts to enter your Minecraft login details, then press play and wait for all of the files to download (aproxx. 10 minutes).  (You must do this for each specific version to play that version!)
```
Comment (Flopgop) [inline README.md:34]: Should probably use the full word here instead of just "perms"

File: README.md [hunk 2]
```diff
@@ -1,100 +1,113 @@
...
 
 And boom! You have successfully installed QuestCraft on your VR headset! To play, press the **Play** button and wait for it to load; depending on the headset's age, it might take a while.
 
-## Supported Renderers (Take note when installing mods!)
+### DO NOT ENABLE SEATED MODE!
+
+**NOTE: You may encounter crashes when downloading your selected MC version, ESPECIALLY on v53 and later. This is because Meta can't fix memory leaks, for some reason. Don't fret; launch the app again, press play again, and the download will continue where it left of before the crash. If it continues, please keep trying. If, after 4 attempts, you still are not loaded in, please visit the support channel of the QuestCraft Discord (discord.gg/QuestCraft).**
```
Comment (Flopgop) [inline README.md:40]: "left of" instead of "left off"

File: README.md [hunk 3]
```diff
@@ -1,100 +1,113 @@
...
+*(For more information, please read the **[Discord announcement.](https://discord.com/channels/820767484042018829/820788932123688960/1138300818064416801)**)*
+
+         
 # QuestCraft
 ![QC](/QCSimple3.jpg)
 
-QuestCraft, utilizing Vivecraft and Pojlib, is a standalone port of Minecraft: Java Edition to Oculus Quest Headsets. With a team of four main developers and 8 third-party developers, over the past year, we have not only gotten the bestselling game, Minecraft: Java Edition running on Android, but we have also worked with third-party developers to bring it into Virtual Reality on the Oculus Quest line of standalone HMDs.
+QuestCraft, utilizing Vivecraft and Pojlib, is a standalone port of Minecraft: Java Edition to Oculus Quest Headsets. With a team of four main developers and 8 third-party developers, over the past year, we have not only gotten Minecraft: Java Edition to run Android, but we have also worked with third-party developers to bring it into Virtual Reality on the Meta Quest line of standalone HMDs.
```
Comment (Flopgop) [inline README.md]: "to run" instead of "to run on"

**Review by Flopgop — CHANGES_REQUESTED** [review]
Misc grammar changes.

**Review by Flopgop — APPROVED** [review]
problems solved 👍

**Comment by DED0026** [comment]
ded approves :trollface:

**Comment by TecEash1** [comment]
Techy approves :trollface:

**Comment by swaggio7** [comment]
I have an idea! May take some time to complete it, though

File: README.md [hunk 4]
```diff
@@ -108,15 +121,22 @@ If you need help with these solutions, please go to the support channel of the Q
...
 
 These controls are subject to change.
+
+## README Contributors:
+
+<a href="https://github.com/QuestCraftPlusPlus/QuestCraft/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=QuestCraftPlusPlus/QuestCraft" />
+</a>
+
+Made with [contrib.rocks](https://contrib.rocks).
```
Comment (TecEash1) [inline README.md:142]: btw u may want to remove this

Comment (TecEash1) [inline README.md:142]: btw u may want to remove this line

``Made with [contrib.rocks](https://contrib.rocks).``

**Review by TecEash1 — CHANGES_REQUESTED** [review]

**Comment by swaggio7** [comment]
> I have an idea! May take some time to complete it, though



Nevermind.

**Review by TecEash1 — APPROVED** [review]

**Comment by KendrickLamar2003** [comment]
Welp, I need to make another PR.

**Pull request closed without merging**